core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/messages/internal/Messages.java

```diff
@@ -55,6 +55,7 @@
 	public static final String ItemAggregate_TOTAL = "ItemAggregate_TOTAL"; //$NON-NLS-1$
 	public static final String ItemAggregate_VARIANCE = "ItemAggregate_VARIANCE"; //$NON-NLS-1$
 	public static final String ItemAggregate_VARIANCEP = "ItemAggregate_VARIANCEP"; //$NON-NLS-1$
+	public static final String ITEM_COLLECTION_DESC = "ITEM_COLLECTION_DESC";
```

Comment:
Is this still relevant? I think the key in the properties file was removed?

Reply:
Good catch, forgot moving the properties file entry.